misc/requirements/requirements-pyinstaller.txt-raw

```diff
@@ -1 +1 @@
-PyInstaller
+git+https://github.com/pyinstaller/pyinstaller.git@develop
```

Comment:
Not sure this is the right thing to be doing here. I want the dependency update robot to notify us when the new release is out but not to upgrade to a new arbitrary commit on the branch. I don't think this will do that. Probably I should just put the commit sha here and subscribe to releases on the repo.

Reply:
Given that the selection of what commit to use is somewhat arbitrary, personally I'd be fine with updating this in a controlled fashion once a week (and reviewing what changed during the dependency upgrade) until PyInstaller 6 is out.

Reply:
I'm not sure I agree with that. Because it's an arbitrary commit I expect us to have to do extra verification of it as it hasn't been through upstream's normal release process. Are the smoke tests good enough to cover all the functionality the installer could break? I don't know but I would rather only do the extra verification I think is required once.
They seem to release every month so I doubt we would be waiting long.

Reply:
I've pinned this to a specific commit in both requirements files for now. Let me know if you really want me to put it back (recompile_requirements.py was updating to the latest develop commit just fine with the version commented on here).